imageproxy.go

```diff
@@ -409,6 +410,14 @@ func (t *TransformingTransport) RoundTrip(req *http.Request) (*http.Response, er
 
 	defer resp.Body.Close()
 
+	if resp.StatusCode >= 400 {
+		if resp.StatusCode == 400 {
+			return nil, errors.ErrCustom.New("Bad Request (400) received when retrieving image.")
```

Comment:
For both of these new errors we should include the URL that failed in the message (for ease of debugging) unless the URL is shown along with the error returned. 

For this one in particular, we could also use `errors.ErrBadRequestReason.New()` to add the statusCode the error.